recipe/conda_build_config.yaml

```diff
@@ -1,9 +1,11 @@
 # cuda 9.2, gcc 7.3
 # cuda 10.0, gcc 7.3
 c_compiler_version:
-  - 7.3
+  - 7.3    # [not osx]
+  - 4.0.1  # [osx]
```

Comment:
We should use here instead clang version 10.0.0, or is there a strong reason to use instead old 4.0.1?

Reply:
Updated with 9b8e8cf